### DISCUSSION TITLE
Add support for WearOS

### DISCUSSION THREAD

**Discussion by SashaMilic** [yungd1plomat/AdvancedSharpAdbClient#111] [General]
This one should be strait forward. I work with WearOS (android based OS for wearables).

**Question:**
Can you update Nuget packet and/or the project to support command to connect to watch using same process and command for connecting but used additional attribute to be able to pair to watch? 

**Explanation:**
The adb command for pairing and connecting is (`adb pair ip:port pairingCode`):
I think everything should be the same as for connect function, but with additional parameter <pairingCode>.

**Example:**
```cs
static AdbClient adbClient;
static DeviceData deviceData;

static void Main(string[] args)
{
    adbClient = new AdbClient();
    adbClient.Pair("127.0.0.1:62001 243559");
    device = adbClient.GetDevices().FirstOrDefault(); // Get first connected device
    ...
}
```

**Comment by wherewhere** [comment]
But it should be
```cs
adbClient.Pair("127.0.0.1:62001", 243559);
```

**Reply by SashaMilic** [reply]
Don't know what is the best way. The only thing I know is the command that pair and connect watch to adb server.
For example:
$: adb pair 127.0.0.1:62001 243559

**Reply by wherewhere** [reply]
https://github.com/SharpAdb/AdvancedSharpAdbClient/blob/main/AdvancedSharpAdbClient/Interfaces/IAdbClient.cs#L312-L327

**Reply by SashaMilic** [reply]
Love it.. Thank you very much.. Great job..!!! Will test it tomorrow..
